libs/shared/form-fields/src/lib/CompanySearchController/CompanySearchController.tsx

```diff
@@ -94,6 +111,34 @@ export const CompanySearchController: FC<Props> = ({
     return options || []
   }
 
+  const getIsatNumbers = (nationalId: string) => {
+    if (!shouldIncludeIsatNumber) return ''
+
+    const companies: Array<any> = data?.companyRegistryCompanies?.data ?? []
+    if (companies.length === 0) return ''
+
+    const filteredCompany = companies.filter((company) => {
+      if (company.nationalId === nationalId) return true
+      return false
+    })
```

Comment:
This can be simplified to
```suggestion
    const filteredCompany = companies.filter((company) => company.nationalId === nationalId )
```